ui/ducks/send/send.js

```diff
@@ -182,9 +182,19 @@ async function estimateGasLimitForSend({
 }) {
   let isSimpleSendOnNonStandardNetwork = false;
 
-  // blockGasLimit may be a falsy, but defined, value when we receive it from
-  // state, so we use logical or to fall back to MIN_GAS_LIMIT_HEX.
-  const blockGasLimit = options.blockGasLimit || MIN_GAS_LIMIT_HEX;
+  // state, so we use logical or to fall back to MIN_GAS_LIMIT_HEX. Some
```

Comment:
I think we want to keep the comment starting on the original line 185 so that the comment block doesn't start with `state,`

Reply:
@darkwing applied in latest commit